tests/core/test_accounts.py

```diff
@@ -434,44 +419,83 @@ def test_eth_account_recover_transaction_from_eth_test(acct, transaction):
     assert acct.recoverTransaction(raw_txn) == expected_sender
 
 
-@pytest.mark.parametrize(
-    'private_key, password, kdf, expected_decrypted_key, expected_kdf',
-    [
+def encrypt_mark_params():
```

Comment:
> tests/core/test_accounts.py:441: in <module>
    web3js_key(),
E   _pytest.warning_types.RemovedInPytest4Warning: Fixture "web3js_key" called directly. Fixtures are not meant to be called directly, are created automatically when test functions request them as parameters. See https://docs.pytest.org/en/latest/fixture.html for more information.

See: https://github.com/pytest-dev/pytest/issues/4112#issuecomment-439039575

Reply:
Minor suggestion: Yep this change seems to make sense, but I generally try not to mix concerns of PR to make sure they get merged nice and easy.
That of course depends on core-dev policies and I'm not sure what's applied here

Reply:
The build is actually failing due to this.

Reply:
Oh I didn't know they were failing on warnings, well done then!

Reply:
```
====================================================================================== ERRORS =======================================================================================
___________________________________________________________________ ERROR collecting tests/core/test_accounts.py ____________________________________________________________________
tests/core/test_accounts.py:441: in <module>
    web3js_key(),
E   _pytest.warning_types.RemovedInPytest4Warning: Fixture "web3js_key" called directly. Fixtures are not meant to be called directly, are created automatically when test functions request them as parameters. See https://docs.pytest.org/en/latest/fixture.html for more information.
================================================================================= warnings summary ==================================================================================
venv/lib/python3.7/site-packages/eth_keys/datatypes.py:105
  /Users/stefanmendoza/Github/Forks/ethereum/eth-account/venv/lib/python3.7/site-packages/eth_keys/datatypes.py:105: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    class BaseKey(ByteString, collections.Hashable):

eth_account/account.py:1
  /Users/stefanmendoza/Github/Forks/ethereum/eth-account/eth_account/account.py:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import (

venv/lib/python3.7/site-packages/rlp/sedes/raw.py:6
  /Users/stefanmendoza/Github/Forks/ethereum/eth-account/venv/lib/python3.7/site-packages/rlp/sedes/raw.py:6: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import Sequence

venv/lib/python3.7/site-packages/rlp/lazy.py:1
  /Users/stefanmendoza/Github/Forks/ethereum/eth-account/venv/lib/python3.7/site-packages/rlp/lazy.py:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import Iterable, Sequence

venv/lib/python3.7/site-packages/attrdict/mixins.py:5
  /Users/stefanmendoza/Github/Forks/ethereum/eth-account/venv/lib/python3.7/site-packages/attrdict/mixins.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import Mapping, MutableMapping, Sequence

-- Docs: https://docs.pytest.org/en/latest/warnings.html
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================== 5 warnings, 1 error in 0.59 seconds ========================================================================
```